docs/modules/configuration/pages/configuring-with-system-properties.adoc

```diff
@@ -1,12 +1,12 @@
-= Configuring with System Properties
+= Configuring with System Properties and Environment Variables
```

Comment:
The configuration override section is meant to just let users know that it is possible. Users could use sys props or environment variables without the need to use a configuration file. This is why it is its own page rather than being embedded.

Reply:
Yes I that is also fine. It could have separate page. I just pointed out that we shouldn't have duplicate parts. I can see you fixed that in https://github.com/hazelcast/hz-docs/pull/336/commits/d6508ff6e8b549b97d3e755ab9a1d76cf1974e7b so count this as resolved.